example_site/src/components/ConfigurationControls.js

```diff
@@ -84,9 +69,20 @@ export default function ConfigurationControls({ isOpen }) {
     return orientation ? 'right' : 'left';
   }
 
-  const onSubmit = async (formData) => {
-    const configuration = getConfiguration(formData);
+  function clickButton (config) {
```

Comment:
Let's give this a slightly more descriptive name - how about `selectSiteConfiguration`?